### PR TITLE
History fix

### PR DESCRIPTION
**Pull request by nikhilaravi**
- added configuration for browserHistory but reverted back to hashHistory so the website is functional. 

- will investigate cloudfront routing in more detail and then try and reimplement browserHistory

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const basename = '/isearch/0.14/index.html';
```
Comment (nelsonic) [inline config.js]: could you not derive this by doing `require('pacakge.json').version.split('.')[2]` to avoid having to change it on every release...?

Comment (nikhilaravi) [inline config.js]: Ah yes could do that. I have added a line in the gulpfile which does something similar but your suggestion might be simpler!

**Comment by nelsonic** [comment]
Travis is taking his sweet time... all those Yanks are online blitzing the poor Travis servers...
(*Code looks good tho...*)

File: config.js (hunk 1 above)
Comment (nelsonic) [inline config.js]: issue created: https://github.com/numo-labs/isearch-ui/issues/154 resolve on next release. 👍

**Pull request merged**